packages/driver/src/cypress.ts

```diff
@@ -397,6 +411,8 @@ class $Cypress {
           return
         }
 
+        this.maybeEmitCypressInCypress('mocha', 'start', args[0])
+
```

Comment:
I don't know why we do `if (this.config('isTextTerminal')) {` before each mocha event is emitted. This indicates we should only emit these events when running in headless run mode, eg on CI, but I don't know why that would be the case.